packages/kernel-6.12/1006-Select-prerequistes-for-gpu-drivers.patch

```diff
@@ -0,0 +1,29 @@
+From 574ee6cf29164ce587f525434611a76e10e4cc6e Mon Sep 17 00:00:00 2001
+From: Martin Harriman <mharrimn@amazon.com>
+Date: Sun, 30 Mar 2025 22:36:38 +0000
+Subject: [PATCH] Select prerequistes for gpu drivers
+
+---
+ drivers/gpu/drm/tiny/Kconfig | 6 ++++++
+ 1 file changed, 6 insertions(+)
+
+diff --git a/drivers/gpu/drm/tiny/Kconfig b/drivers/gpu/drm/tiny/Kconfig
+index f6889f649..98b597e13 100644
+--- a/drivers/gpu/drm/tiny/Kconfig
++++ b/drivers/gpu/drm/tiny/Kconfig
+@@ -85,6 +85,12 @@ config DRM_SIMPLEDRM
+ 	select APERTURE_HELPERS
+ 	select DRM_GEM_SHMEM_HELPER
+ 	select DRM_KMS_HELPER
++	select DRM_DISPLAY_DP_HELPER
++	select DRM_DISPLAY_HELPER
++        select DRM_KMS_HELPER
++	select DRM_SUBALLOC_HELPER
++        select DRM_TTM
++	select DRM_TTM_HELPER
```

Comment:
Why this is a patch as opposed to additional lines in the config file?

Please fix the spacing if the patch is required.

Reply:
It's a patch because these aren't available as independent configs.

Reply:
I will check the status of these Kconfigs, in case they are now menu-visible and can be toggled independently in the new kernel.

Reply:
This is my best guess approximation of the patch we carry in 6.1 to enable the modules we need for GPU drivers in that kernel. I have tried to select the options directly from `config-bottlerocket` without success.